blues/bondorders.py

```diff
@@ -0,0 +1,16 @@
+import os
+import numpy as np
+from rdkit import Chem
```

Comment:
`rdkit` is not actually needed for this function, so it can be removed from imports.